### PR TITLE
chart bump version 026

### PR DESCRIPTION
**Pull request by rodrigo-findhotel**
- Readme file updated
- `appVersion` value changed in the `Chart.yaml` file to keep it equal to the `version` value
- Added new values on the probes in the `deployment.yaml` file
- Docs folder created with some documentations
- Deploy script updated with new bump version

### CODE REVIEW

File: charts/cf-review-env/templates/deployment.yaml [hunk 1]
```diff
@@ -46,26 +46,31 @@ spec:
...
+            successThreshold: {{ .Values.readinessProbe.successThreshold | default "2" }}
+            failureThreshold: {{ .Values.readinessProbe.failureThreshold | default "5" }}
           startupProbe:
             httpGet:
               path: {{ .Values.startupProbe.httpGet.path | default "/" }}
               port: {{ .Values.deployment.ports.containerPort }}
               scheme: {{ .Values.startupProbe.httpGet.scheme | default "HTTP" }}
-            periodSeconds: {{ .Values.startupProbe.periodSeconds | default "5" }}
-            failureThreshold: {{ .Values.startupProbe.failureThreshold | default "60" }}   
+            initialDelaySeconds: {{ .Values.startupProbe.initialDelaySeconds | default "10" }}
```
Comment (victorpoluceno) [inline charts/cf-review-env/templates/deployment.yaml:69]: There is no need to have initialDelaySecond for the startup probe, it should start as soon as possible.

File: charts/cf-review-env/templates/deployment.yaml [hunk 2]
```diff
@@ -46,26 +46,31 @@ spec:
...
+            periodSeconds: {{ .Values.startupProbe.periodSeconds | default "15" }}
```
Comment (victorpoluceno) [inline charts/cf-review-env/templates/deployment.yaml:70]: 15 second for the peridoSeconfs is too high, I think 5 seconds is better.

File: charts/cf-review-env/templates/deployment.yaml [hunk 3]
```diff
@@ -46,26 +46,31 @@ spec:
...
+            timeoutSeconds: {{ .Values.startupProbe.timeoutSeconds | default "5" }}
+            successThreshold: {{ .Values.startupProbe.successThreshold | default "2" }}
+            failureThreshold: {{ .Values.startupProbe.failureThreshold | default "10" }}
```
Comment (victorpoluceno) [inline charts/cf-review-env/templates/deployment.yaml:73]: I think if we have a periodSeconds of 5, this failureThreshold should be bigger. I think 30 is a good number, so 2,5 minutes to start.

File: charts/cf-review-env/templates/deployment.yaml [hunk 4]
```diff
@@ -46,26 +46,31 @@ spec:
...
             httpGet:
               path: {{ .Values.startupProbe.httpGet.path | default "/" }}
               port: {{ .Values.deployment.ports.containerPort }}
               scheme: {{ .Values.startupProbe.httpGet.scheme | default "HTTP" }}
-            periodSeconds: {{ .Values.startupProbe.periodSeconds | default "5" }}
-            failureThreshold: {{ .Values.startupProbe.failureThreshold | default "60" }}   
+            initialDelaySeconds: {{ .Values.startupProbe.initialDelaySeconds | default "10" }}
+            periodSeconds: {{ .Values.startupProbe.periodSeconds | default "15" }}
+            timeoutSeconds: {{ .Values.startupProbe.timeoutSeconds | default "5" }}
+            successThreshold: {{ .Values.startupProbe.successThreshold | default "2" }}
```
Comment (victorpoluceno) [inline charts/cf-review-env/templates/deployment.yaml]: 1 sucess is already enough.

**Review by victorpoluceno — COMMENTED** [review]
Nice changes in the docs. I left some comments related to the startupProbe numbers.

File: charts/cf-review-env/templates/deployment.yaml (hunk 3 above)
Comment (rodrigo-findhotel) [reply charts/cf-review-env/templates/deployment.yaml:73]: I saw some articles about `startupProbe` and in it the` failureThreshold` they said the number was better than `readinessProbe` and` livenessProbe`.

Comment (victorpoluceno) [reply charts/cf-review-env/templates/deployment.yaml:72]: hmm, that is weird. For me it doesn't make sense, do they say why?

**Review by victorpoluceno — APPROVED** [review]

File: charts/cf-review-env/templates/deployment.yaml (hunk 3 above)
Comment (rodrigo-findhotel) [reply charts/cf-review-env/templates/deployment.yaml:72]: Sorry, I invert the idea and on the articles, the correct one is on the `startupProbe` be bigger or equal than `livenessProbe` and `readinessProbe`.

**Pull request merged**